pkg/ctl/get/cluster_test.go

```diff
@@ -0,0 +1,17 @@
+package get
```

Comment:
this is the only verb command that I added unit test for each resource :disappointed_relieved: